media/Jsontracer/index.js

```diff
@@ -124,7 +124,7 @@ function changeSlider(inputValue, inputMax, inputMin) {
 
   // set delay
   slider.disabled = true;
-  setTimeout((slider.disabled = false), 100);
+  setTimeout(() => (slider.disabled = false), 100);
```

Comment:
This change is necessary for this PR?

Reply:
That's not.
It was too small to divide, so I put it in together. There is no error in the operation, but we changed it for uniformity.

Reply:
If so, please separate this diff :)
There is no problem that PR is so small :D

Reply:
Okay! Thank you 👍